# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libkarma/hotfix.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libkarma-0.1.0/tools/Makefile.vanilla	2007-05-19 12:31:05.000000000 +0000
+++ libkarma-0.1.0/tools/Makefile	2007-05-19 12:33:12.000000000 +0000
@@ -19,7 +19,8 @@
 
 CC=gcc
 CFLAGS+=-Wall -pedantic -g
-LDFLAGS+=-ltag_c -lz -L $(LIBDIR)
+LDFLAGS+=-L $(LIBDIR)
+LDLIBS+=-ltag_c -lz
 
 OBJS=$(SRCS:.c=.o)
 TOOLS=riocp chprop karma_helper
@@ -31,16 +32,16 @@
 	@echo Linking tools with the static lib: $(LIBKARMA)
 
 riocp: riocp.c $(OBJS) $(LIBKARMA)
-	$(CC) $(INCLUDES) $(LDFLAGS) $(CFLAGS) $@.c -o $@ $(OBJS) $(LKARMA) 
+	$(CC) $(INCLUDES) $(LDFLAGS) $(CFLAGS) $@.c -o $@ $(OBJS) $(LKARMA) $(LDLIBS)
 
 chprop: chprop.c $(LIBKARMA)
-	$(CC) $(INCLUDES) $(LDFLAGS) $(CFLAGS) $@.c -o $@  $(LKARMA)
+	$(CC) $(INCLUDES) $(LDFLAGS) $(CFLAGS) $@.c -o $@  $(LKARMA) $(LDLIBS)
 
 playlist_show: playlist_show.c $(LIBKARMA)
-	$(CC) $(INCLUDES) $(LDFLAGS) $(CFLAGS) $@.c -o $@  $(LKARMA)
+	$(CC) $(INCLUDES) $(LDFLAGS) $(CFLAGS) $@.c -o $@  $(LKARMA) $(LDLIBS)
 
 karma_helper: karma_helper.c
-	$(CC) $(INCLUDES) $(CFLAGS) $@.c -o $@ -lusb
+	$(CC) $(INCLUDES) $(CFLAGS) $@.c -o $@ -lusb $(LDLIBS)
 
 install:
 	../install-sh -m 0755 -d $(DEST)/bin
